mta-sts.md

```diff
@@ -480,6 +480,40 @@ Recipients should also prefer to update the HTTPS policy body before updating
 the TXT record; this ordering avoids the risk that senders, seeing a new TXT
 record, mistakenly cache the old policy from HTTPS.
 
+## Policy Delegation
+
+Domain owners commonly delegate SMTP hosting to a different organization, such
+as an ISP or a Web host. In such a case, they may wish to also delegate the
+MTA-STS policy to the same organization which can be accomplished with two
+changes.
+
+First, the Policy Domain must point the `_mta-sts` record, via CNAME, to
+the `_mta-sts` record maintained by the hosting organization. This allows the
+hosting organization to control update signaling.
+
+Second, the Policy Domain must point the "well-known" policy location to the
+hosting organization. This can be done either by redirecting the `mta-sts` host to a
+host or CNAME specified by the hosting organization and by giving the hosting
+organization a TLS certificate which is valid for that host, or by setting up a
```

Comment:
> This can be done either by redirecting the `mta-sts` host to a
 > host or CNAME specified by the hosting organization and by giving the hosting
 > organization a TLS certificate which is valid for that host

The 'redirect' may confuse with 302 redirects, and suggest a similar sentence below:
\=
This can be done by delegating the `mta-sts` sub-domain to the hosting organization, and by giving the hosting organization a TLS certificate which is valid for `mta-sts` subdomain, or by ...
\=

Reply:
It's actually interesting that you can (I think) do a few different things herE:

1. Make mta-sts a CNAME to a host specified by the hosting org. Probably easiest. 
2. Create an mta-sts subdomain with NS pointed to those provided by the hosting org. This works, since you can have an apex record that's an A or CNAME, but I don't know why you'd bother.

For clarity I'm suggesting option 1, but either works.